nupic/encoders/scalar.capnp

```diff
@@ -0,0 +1,20 @@
+@0xfa7d16f86048a6e4;
+
+struct ScalarEncoderProto {
+  w @0 :Int32;
+  minval @1 :Float32;
+  maxval @2 :Float32;
+  periodic @3 :Bool;
+  n @4 :Int32;
+  radius @5 :Float32;
+  resolution @6 :Float32;
+  name @7 :Text;
+  verbosity @8 :Int8;
+  clipInput @9 :Bool;
+
+  halfwidth @10 :Int32;
```

Comment:
It would be nice to have descriptions of these attributes. I don't know why `halfwidth` is needed for example. If you don't know why some of these exist then we should figure it out.